readme.md

```diff
@@ -145,6 +146,9 @@ If you have used any of these styleguides, please add a comment if you can! The
 ### Riot
 * [RiotJS Style Guide](https://github.com/voorhoede/riotjs-style-guide#readme) - Opinionated RiotJS Style Guide for teams by [@deVoorhoede](https://twitter.com/devoorhoede).
 
+### Vue
+* [VueJs Style Guide](https://vuejs.org/v2/style-guide/) - The official style guide for Vue-specific code.
```

Comment:
Thanks, please write it as Vue like they do:
```diff
-VueJs Style Guide
+Vue Style Guide
```

Reply:
@hugovk  
Thank you for the review!
I updated.